### DISCUSSION TITLE
Unable to run docker compose up command in windows

### DISCUSSION THREAD

**Discussion by sandeep1437** [apiman/apiman#2493] [Help]
Hi Everyone,

I'm new to apiman. While trying to run initial command in windows docker-compose -f docker-compose.setup.yml up after extracting 3.0.0 version zip file I'm gettting this kind of error. can anyone explain me what's my mistake is ?
<img width="847" alt="image" src="https://user-images.githubusercontent.com/58900240/227324602-dd21cd17-899a-4707-a0c7-77c49777b718.png">

**Comment by msavy** [comment]
> Hi Everyone,
> 
> I'm new to apiman. While trying to run initial command in windows docker-compose -f docker-compose.setup.yml up after extracting 3.0.0 version zip file I'm gettting this kind of error. can anyone explain me what's my mistake is ?
> <img width="847" alt="image" src="https://user-images.githubusercontent.com/58900240/227324602-dd21cd17-899a-4707-a0c7-77c49777b718.png">
> 

Did you follow the instructions in the readme about Windows? @martinpurvis provided some steps.

**Reply by sandeep1437** [reply]
Yes I have read the readme but I was not able to understand where to put the `set PWD=%CD%`. as per my understanding have to add it on working_dir in the docker-compose.setup.yml but when ever I did this and run docker-compose.setup.yml up command I'm getting the above error. Even I have mentioned my path in the place of %CD% where my apiman zip file was extracted.

<img width="833" alt="image" src="https://user-images.githubusercontent.com/58900240/227707749-4f4a7228-13fe-4365-8236-c87c28f558b1.png">

**Comment by msavy** [comment]
That is not correct. You run `set PWD=%CD%` in your terminal before executing any Docker Compose commands.

Wherever possible, please do not use screenshots of code, errors or stack traces; instead it is best to use code blocks. This is more accessible for search, people with visual impairments, and easier for developers to copy and paste.

**Reply by sandeep1437** [reply]
Thank you Savy, Is there any video tutorials to learn how to use API umbrella ?

**Reply by msavy** [reply]
I think that is a different project entirely, sorry.

**Reply by sandeep1437** [reply]
I'm getting `404 page not found` after running docker-compose up command and trying to access this url `http://apiman.local.gd:8080/apimanui`
